crypto/src/conversation.rs

```diff
@@ -793,11 +840,91 @@ pub mod tests {
                 .decrypt_message(&encrypted_message, &alice_backend)
                 .await
                 .unwrap()
+                .0
                 .unwrap();
             assert_eq!(original_message, roundtripped_message.as_slice());
         }
     }
 
+    pub mod delay {
```

Comment:
Add tests with combinations of (self_index, epoch, total_members) with values from `usize::MIN` to `usize::MAX`